pyrasterframes/src/main/python/setup.py

```diff
@@ -140,6 +140,7 @@ def dest_file(self, src_file):
     author_email='info@astraea.earth',
     license='Apache 2',
     url='https://rasterframes.io',
+    python_requires=">=3.7",
```

Comment:
I would guess that might be too strict. On my laptop my default is python 3.6. I am not sure how far back to allow.